Holy_locket.WebAPI/Controllers/DoctorsController.cs

```diff
@@ -0,0 +1,139 @@
+﻿using Holy_locket.WebAPI.TestModels;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using System.IO;
+using System.Numerics;
+using System.Reflection.PortableExecutable;
+using System.Text.Json;
+
+namespace Holy_locket.WebAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    //[Route("Doctors")]
```

Comment:
don't push comments, if you'll need to update controller further please add code then